tests/components/rfxtrx/test_config_flow.py

```diff
@@ -0,0 +1,47 @@
+"""Test the Tado config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.rfxtrx import DOMAIN
+
+from tests.common import MockConfigEntry
+
+
+async def test_import(hass):
+    """Test we can import."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_IMPORT},
+        data={"host": None, "port": None, "device": "/dev/tty123", "debug": False},
+    )
+
+    assert result["type"] == "create_entry"
+    assert result["title"] == "RFXTRX"
+    assert result["data"] == {
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)